### PR TITLE
cgroup: whitelist inaccessible devices for "auto" and "closed" DevicePolicy

### PR DESCRIPTION
**Pull request by alepuccetti**

### CODE REVIEW

File: units/systemd-nspawn@.service.in [hunk 1]
```diff
@@ -25,15 +25,8 @@ TasksMax=16384
...
-DeviceAllow=/dev/full rwm
-DeviceAllow=/dev/random rwm
-DeviceAllow=/dev/urandom rwm
-DeviceAllow=/dev/tty rwm
+DevicePolicy=closed
 DeviceAllow=/dev/net/tun rwm
-DeviceAllow=/dev/pts/ptmx rw
 DeviceAllow=char-pts rw
 DeviceAllow=/run/systemd/inaccessible/chr rwm
 DeviceAllow=/run/systemd/inaccessible/blk rwm
```
Comment (poettering) [inline units/systemd-nspawn@.service.in]: these last two lines should go, too, no?

**Comment by alepuccetti** [comment]
Patch updated

**Comment by poettering** [comment]
something is skewed with the two patches. the removal if the inaccessible device nodes from the bus call and from the nspawn unit file should really happen in the same patch, but you put them in different patches? LOoks like a mistake...

If you ask me, just merge the two patches into one!

**Comment by alepuccetti** [comment]
I will rebase this on the current master (with the freshly merged patch) and updated

**Comment by alepuccetti** [comment]
This should make more sense

**Comment by poettering** [comment]
lgtm.

**Pull request merged**